shared/middleware/helpers/storage_apps.rb

```diff
@@ -13,10 +13,12 @@ def initialize(storage_id)
   end
 
   def create(value, ip_address)
+    timestamp = Time.now
```

Comment:
Whoops, I totally didn't mean to switch from `DateTime` to `Time`.  Good catch!  I will fix.